README.md

```diff
@@ -21,31 +21,43 @@ Currently the docker-compose performance isn't very good. Thus, for local dev, i
 
 We need to install airflow.
 
-1. First, we should create virtual environment with pyenv:
+1. First, we need to install poetry:
 
-```
-    export PYTHON_VERSION=3.8.13
-    pyenv install $PYTHON_VERSION
-    pyenv global $PYTHON_VERSION
-    pyenv virtualenv $PYTHON_VERSION workflows
-    pyenv activate workflows
-```
+    ```
+    curl -sSL https://install.python-poetry.org | python3 -
 
-3. Set airflow home directory:
+    ```
 
-```
-export AIRFLOW_HOME=/path/to/cloned/repo
-```
-
-4. Install all required dependencies for a project. We need 3 requirements files, when we're running the project locally. First file (requirements.py) has all additional dependencies required to run the tasks correctly (connecting to ftputil, boto3 and etc.), the second one installs requirememnts for testing, such as pytest, the third- (requirements_airflow.py) installs Airflow itself and constraints needed for it. The the third file is not needed when we're running the procject on Docker, because we're using Apache Airflow Docker image:
-   `pip install -r requirements.txt -r requirements-test.txt -r requirements-airflow.txt`
-5. Run Airflow. Airflow comes with the `standalone` command. This should not be used as it seems to use the SequentialExecutor by default. To have our local config being use, we must run all services together :
+3. Set airflow home directory:
 
-```
-    airflow webserver
-    airflow triggerer
-    airflow scheduler
-```
+    ```
+    export AIRFLOW_HOME=/path/to/cloned/repo
+    ```
+4. Activate poetry virtual environment:
+    ```
+    poetry shell
+    ```
+    If you are using incorrect python version, you will get an error:
+    ```
+    current Python version (X.Y.Z) is not allowed by the project (^X.Y).
+    Please change python executable via the "env use" command.
+    ```
+    You can easily fix it with the following command:
+    ```
+    poetry env use X.Y
+    ```
+    The correct python version can be found in pyptoject.toml file
```

Comment:
hmm is this copy paste from somewhere else? we should specify the version that the project uses.

Reply:
No, I wrote by myself. The python version might change with every update, am I right?

Reply:
if we change the python version we should update the docs, let's keep the version that we would like to run it